generators/webforms/templates/manifest.dnn

```diff
@@ -15,7 +15,7 @@
       <releaseNotes src="ReleaseNotes.txt" />
       <azureCompatible>true</azureCompatible>
       <dependencies>
-        <dependency type="CoreVersion">09.08.00</dependency>
+        <dependency type="CoreVersion">09.11.00</dependency>
```

Comment:
@ArielBlanco1990 There should be a variable to use for this, instead of hard-coding the version number here. Please follow the pattern that was in place before this update. It previously used `dnnBuildVersionShort`. If that doesn't work and there's not a `dnnBuildVersionLong` or something, then we should have it created. Ideally, this update should be done in as few places as possible when we need to do it again in the future.

Reply:
@WillStrohl I followed your instructions and solved this problem. There was no `dnnBuildVersionLong `but there was `dnnBuildVersion `which is what we need. I replaced `<%= dnnBuildVersionShort %>` and `<%= dnnBuildVersion %>` in each case.